core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -71,13 +75,18 @@ object WorkspaceMigrationMonitor {
                                  storageTransferService: GoogleStorageTransferService[IO])
 
 
-  type MigrateAction[T] = ReaderT[OptionT[IO, *], MigrationDeps, T]
+  type MigrateAction[A] = ReaderT[OptionT[IO, *], MigrationDeps, A]
```

Comment:
What does the change from `T` to `A` do?

Reply:
Nothing, it's a style change for consistency.

Reply:
Works for me. Would it be worth changing the other `T` values as well then?

Reply:
Fix as we go I think :)

Reply:
On you mean in this PR? Yeah sure!

Reply:
Yeah, I meant in this PR :D, since we're already changing some values.